firebase-functions/firebase.json

```diff
@@ -18,5 +14,17 @@
         "npm --prefix \"$RESOURCE_DIR\" run build"
       ]
     }
-  ]
+  ],
+  "emulators": {
+    "functions": {
+      "port": 5001
```

Comment:
I wonder if the port is static.

Reply:
If you mean if it has to be that way, the answer is no. 
These are the default ports for each of the parts of the emulator that the CLI sets and that documentation refers to, but we could change it to something different and it should still work.
You have some personal grievance with these numbers in particular? :)